spec/controllers/pages_controller_spec.rb

```diff
@@ -24,4 +24,21 @@ def index
       expect(response).to render_template(layout: false)
     end
   end
+
+  describe 'content expiry' do
```

Comment:
ah ok, so perhaps we should write a test for a controller that does *not* skip this?